android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -23,18 +23,52 @@
 
     private String      adUnitId;
     private MaxAdFormat adFormat;
-
-    private String  placement;
-    private String  customData;
-    private Boolean adaptiveBannerEnabled;
-    private Boolean autoRefresh;
+    private String      placement;
+    private String      customData;
+    private Boolean     adaptiveBannerEnabled;
+    private Boolean     autoRefresh;
 
     public AppLovinMAXAdView(final Context context)
     {
         super( context );
         this.reactContext = (ThemedReactContext) context;
     }
 
+    void setAdUnitId(final String value)
+    {
+        // Ad Unit ID must be set prior to creating MAAdView
+        if ( adView != null )
+        {
+            AppLovinMAXModule.e( "Attempting to set Ad Unit ID " + adUnitId + " after MAAdView is created" );
```

Comment:
`MAAdView` is an iOS class. `MaxAdView` is the Android one.

Reply:
Updated all.